java.md

```diff
@@ -15,3 +15,7 @@ Javadoc
 -------
 
 In addition to the [Google standards on javadoc](https://google.github.io/styleguide/javaguide.html#s7-javadoc) **do not** use the `@author` tag as this is unwanted noise that provides no advantage over version control history.
+
+Checkstyle
+----------
+You can download the [CheckStyle configuration](http://repository.aws.chdev.org:8081/artifactory/libs-snapshot-local/uk/gov/companieshouse/java-checkstyle-config/) for Companies House developments.
```

Comment:
Navigating to http://repository.aws.chdev.org:8081/artifactory/libs-snapshot-local/uk/gov/companieshouse/java-checkstyle-config in a browser shows the directory contents, but what file holds the actual config (one of the XML files?) and what should a developer do with that file (reference it in the IDE?). Perhaps some further instructions should be added or am I missing something?

Reply:
I have replaced that link with one to the README in the https://github.com/companieshouse/java-checkstyle-config repo. I'm think that the instructions for use in IDE should be in that repo.

Reply:
👍  makes sense. Thanks.